### ISSUE TITLE
Show breakpoints

### ISSUE THREAD

**Issue by fredrikekre** [JuliaDebug/Debugger.jl#88]
Breakpoints that are added with e.g. `@breakpoint` and `breakpoint` are not visible in the source. Would be nice to show breakpoints.

**Comment by KristofferC** [comment]
How about a circle in front of the line number?

```
 1
●2
 3
```

or something?

**Comment by fredrikekre** [comment]
If you make it red :)

**Comment by pfitzseb** [comment]
How about
```
 1
 ●
 3
```
for active breakpoints and
```
 1
 ◌
 3
```
for inactive breakpoints?

**Comment by KristofferC** [comment]
If we are worried about the extra initial space in front of the line numbers being unused we could use it as a mark for the current line as well. It is nice to not only have a color to show the current line.

```
 1
>2
 3
```

**Comment by pfitzseb** [comment]
Yeah, I've wanted that too (mostly because the colors don't work when copy-pasting an example but it still is interesting on what line you are).
